app/assets/stylesheets/mappings.scss

```diff
@@ -75,4 +75,7 @@ div#map_from_concept_details_table, div#map_to_concept_details_table {
 #concept_mappings_table {
   width: 100%;
 }
+#concept_mappings_table .chip_button_container_clickable{
+  white-space: nowrap;
```

Comment:
what do you do if the concept label is too long, the wrapping was done to prevent that.

Reply:
Okay